src/env.d.ts

```diff
@@ -1 +1,2 @@
+/// <reference path="../.astro/types.d.ts" />
```

Comment:
Committed this by mistake, got autogenerated on my side for some reason. Not sure if its a big deal to keep it out?